File: Ciclo2/Unidad1/EjerciciosSesión4/src/Descuento.java
import javax.swing.*;

public class Descuento {
    public static void main(String[] args) {
        double valor_compra, descuento, total_compra;
        int numero;
        valor_compra = Double.parseDouble(JOptionPane.showInputDialog( "Ingrese el valor de la compra:"));
        numero = Integer.parseInt(JOptionPane.showInputDialog("Ingrese un numero:"));
        if (numero < 74){
            descuento = valor_compra * 0.15;
        }else{
            descuento = valor_compra * 0.2;
        }
        total_compra = valor_compra - descuento;
        JOptionPane.showMessageDialog(null, "El valor descontado es: $"+descuento+", el valor total es: $"+total_compra);
    }
}

File: Ciclo1/Unidad1/ejercicios/descuento.py
def run():
    print('Calcular 15% de decuescuento')
    valor = int(input('Ingrese el valor de la compra: '))
    descuento = valor * 0.15
    valor_total = valor + descuento
    print(
        f'El 15% de {valor} es {descuento}\n El valor total a pagar es {valor_total}')


if __name__ == "__main__":
    run()
